client/package.json

```diff
@@ -3,7 +3,10 @@
   "version": "0.17.0",
   "license": "MIT",
   "description": "cellxgene is a web application for the interactive exploration of single cell sequence data.",
-  "repository": "https://github.com/chanzuckerberg/cellxgene",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/chanzuckerberg/cellxgene.git"
```

Comment:
we should probably update this to explorer

Reply:
Good call thank you! Updated 👍 !